### ISSUE TITLE
package conflict - consider upgrading utoipa

### ISSUE THREAD

**Issue by cameronbraid** [LeonHartley/Coerce-rs#11]
```
 --> /home/cameronbraid/.cargo/registry/src/github.com-1ecc6299db9ec823/coerce-0.8.3/src/remote/api/system/mod.rs:35:76
    |
35  |             .merge(SwaggerUi::new("/swagger").url("/api-doc/openapi.json", ApiDoc::openapi()))
    |                                               ---                          ^^^^^^^^^^^^^^^^^ expected struct `utoipa::openapi::OpenApi`, found a different struct `utoipa::openapi::OpenApi`
    |                                               |
    |                                               arguments to this function are incorrect
    |
    = note: struct `utoipa::openapi::OpenApi` and struct `utoipa::openapi::OpenApi` have similar names, but are actually distinct types
note: struct `utoipa::openapi::OpenApi` is defined in crate `utoipa`
   --> /home/cameronbraid/.cargo/registry/src/github.com-1ecc6299db9ec823/utoipa-2.4.2/src/openapi.rs:36:1
    |
36  | / builder! {
37  | |     /// # Examples
38  | |     ///
39  | |     /// Create [`OpenApi`] using [`OpenApiBuilder`].
...   |
114 | |     }
115 | | }
    | |_^
note: struct `utoipa::openapi::OpenApi` is defined in crate `utoipa`
   --> /home/cameronbraid/.cargo/registry/src/github.com-1ecc6299db9ec823/utoipa-3.0.1/src/openapi.rs:37:1
    |
37  | / builder! {
38  | |     /// # Examples
39  | |     ///
40  | |     /// Create [`OpenApi`] using [`OpenApiBuilder`].
...   |
115 | |     }
116 | | }
    | |_^
    = note: perhaps two different versions of crate `utoipa` are being used?
note: associated function defined here
   --> /home/cameronbraid/.cargo/registry/src/github.com-1ecc6299db9ec823/utoipa-swagger-ui-3.0.2/src/lib.rs:226:12
    |
226 |     pub fn url<U: Into<Url<'static>>>(mut self, url: U, openapi: OpenApi) -> Self {
    |            ^^^
    = note: this error originates in the macro `builder` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0308`.
error: could not compile `coerce` due to previous error
```

if I point cargo at my locally checked out coerce with utoipa upgraded to version 3 this goes away - is that the correct fix ?

**Comment by cameronbraid** [comment]
I should note the above happens when building my app with coerce as a versioned dependency.  And to clarity, if I change my app's cargo.toml to use a path instead of version, and point it at my coerce checkout with upgradede utopia, I am able to compile.

**Comment by LeonHartley** [comment]
Cheers @cameronbraid, I've just updated `utoipa` and `utoipa-swagger-ui` and published 0.8.4.

**Comment by LeonHartley** [comment]
Great news!
